src/Controls/src/Core/ListView.cs

```diff
@@ -606,7 +606,7 @@ internal override void OnIsPlatformEnabledChanged()
 
 		void OnCommandCanExecuteChanged(object sender, EventArgs eventArgs)
 		{
-			RefreshAllowed = RefreshCommand.CanExecute(null);
+			RefreshAllowed = RefreshCommand != null && RefreshCommand.CanExecute(null);
```

Comment:
Fix NRE in ListView Command
https://github.com/xamarin/Xamarin.Forms/pull/14580